integration/deploy_manifest_test.go

```diff
@@ -0,0 +1,322 @@
+//go:build integration
+// +build integration
+
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/registry"
+)
+
+func TestDeployFromManifest(t *testing.T) {
+	ctx := context.Background()
+	oktetoPath, err := getOktetoPath(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	const (
+		gitRepo          = "git@github.com:okteto/go-getting-started.git"
+		repoDir          = "go-getting-started"
+		manifestFilename = "okteto.yml"
+		chartDir         = "chart"
+		chartFilename    = "Chart.yaml"
+		templateFilename = "k8s.yaml"
+		releaseName      = "hello-world"
+		manifestContent  = `build:
```

Comment:
I believe you can add a newline after the `=` or the ` and it will make the entire manifest easier to read.